deps/CMakeLists.txt

```diff
@@ -0,0 +1,78 @@
+# ########################################################################
+# Copyright 2016 Advanced Micro Devices, Inc.
+# ########################################################################
+# Helper cmake script to automate building dependencies for rocblas
+# This script can be invoked manually by the user with 'cmake -P'
+
+# The ROCm platform requires Ubuntu 16.04 or Fedora 24, which has cmake 3.5
+cmake_minimum_required( VERSION 3.5 )
+
+list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/../cmake )
+
+# Consider removing this in the future
+# It can be annoying for visual studio developers to build a project that tries to install into 'program files'
+if( WIN32 AND CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT )
+  set( CMAKE_INSTALL_PREFIX "${PROJECT_BINARY_DIR}/package" CACHE PATH "Install path prefix, prepended onto install directories" FORCE )
+endif( )
+
+# This has to be initialized before the project() command appears
+# Set the default of CMAKE_BUILD_TYPE to be release, unless user specifies with -D.  MSVC_IDE does not use CMAKE_BUILD_TYPE
+if( NOT DEFINED CMAKE_CONFIGURATION_TYPES AND NOT DEFINED CMAKE_BUILD_TYPE )
+  set( CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build, options are: None Debug Release RelWithDebInfo MinSizeRel." )
+endif()
+
+# The superbuild does not build anything itself; all compiling is done in external projects
+project( rocblas-dependencies NONE )
+
+option( BUILD_BOOST "Download and build boost library" ON )
```

Comment:
will this dependency be  OFF  if the user wants to build only library and verification code, not library + verification code + tests + benchmarks?